domains/etercore.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "ramirosanchezaranda",
+      "email": "etercoredesignweb@gmail.com"
+    },
+    "record": {
+      "CNAME": "etercore.vercel.app"
```

Comment:
Make sure to also include a Vercel verification record in another file. You can find out more information on this at the docs: https://docs.is-a.dev/guides/vercel